src/backend/rest.ts

```diff
@@ -261,6 +266,10 @@ export default class RestApiBackend implements Backend {
         'This Checkmk data source is only compatible with Checkmk Cloud and Checkmk MSP, but you are trying to connect to another edition.'
       );
     }
+    // CSE is never supported
+    if (checkmkEdition === 'cse') {
+      throw new Error('This Checkmk data source is not compatible with Checkmk Cloud (SaaS).');
```

Comment:
i think we should rephrase this, as we have not checkmk datasource that is compatible with "Checkmk Cloud (SaaS)". maybe: "Can not query data from Checkmk Cloud (SaaS)."